.github/auto_assign.yml

```diff
@@ -0,0 +1,35 @@
+# Set to true to add reviewers to pull requests
```

Comment:
Maybe add a short comment at the top saying that this is a config file for the [insert name of github app here] bot whose purpose is to auto-assign a reviewer in order to better help distribute reviews across the team, and familiarise developers with parts of the engine they may not typically deal with.

Reply:
done